### PR TITLE
IDE-716 "Collapse All" option for tree elements

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/scite.h [hunk 1]
```diff
@@ -1148,20 +1148,11 @@ class CSciTeExtension
         return true;
     }
 
-    void FoldAll() 
+    void FoldAll(bool expanding = false)
```
Comment (GordonSmith) [inline wlib/scite.h]: `expanding` is not used - how can this work?

Comment (dehilsterlexis) [reply wlib/scite.h]: It is used. I added "Fold All" and "Unfold All". See line 646 of ECLDlgBuilder.cpp: https://github.com/hpcc-systems/eclide/blob/b85053688fb11e94001891f0fa058d585ad1dc37/eclide/EclDlgBuilder.cpp#L646

Comment (GordonSmith) [reply wlib/scite.h]: So it is - I didn't look far enough down...

**Pull request merged**